fe/fe-core/src/test/java/com/starrocks/sql/plan/AggregateTest.java

```diff
@@ -118,7 +118,10 @@ public void testHaving2() throws Exception {
     public void testGroupByNull() throws Exception {
         String sql = "select count(*) from test_all_type group by null";
         String plan = getFragmentPlan(sql);
-        assertContains(plan, "<slot 11> : NULL");
+        // null will be pruned from grouping keys by PruneGroupByKeysRule
```

Comment:
> for sum functions , will you also remove the group by columns?

1. for this cases, the previous implementation is wrong, `select count(*) from test_all_type` and `select count(*) from test_all_type group by null` are not equivalent.I have fixed.
2. for other cases, this rule doesn't care about the type of agg functions, so it will remove group by columns if it can.